backend/infrahub/core/query/diff.py

```diff
@@ -644,7 +644,7 @@ async def query_init(self, db: InfrahubDatabase, **kwargs):
                     r_node.branch = diff_rel.branch DESC,
                     r_prop.from DESC,
                     r_node.from DESC
-                WITH node, prop, head(collect(path)) AS latest_path
+                WITH node, prop, type(r_prop) AS r_prop_type, type(r_node) AS r_node_type, head(collect(path)) AS latest_path
```

Comment:
this update is to allow including multiple paths across the same nodes if those paths are of different types, such as IS_RELATED and HAS_SOURCE/HAS_OWNER